service/src/main/java/com/epam/esm/dto/CertificateRequestFieldDTO.java

```diff
@@ -0,0 +1,34 @@
+package com.epam.esm.dto;
+
+import com.epam.esm.validator.RequestField;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Positive;
+import javax.validation.constraints.Size;
+import java.io.Serializable;
+import java.math.BigDecimal;
+
+@Setter
+@Getter
+@RequiredArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+@RequestField
+public class CertificateRequestFieldDTO implements Serializable {
+    @Size(max = 50)
+    private String name;
+
+    @Size(max = 255)
+    private String description;
+
+    @Positive
+    private BigDecimal price;
+
+    @Max(value = 150)
+    @Min(value = 1)
+    private Integer duration;
```

Comment:
int

Reply:
I cannot do this because I am checking objects for null